### DISCUSSION TITLE
Hello, could you please help me check the reason for the 00.train error in iter. 00000 1 calculated by dpgen simplify?

### DISCUSSION THREAD

**Discussion by XuFanffei** [deepmodeling/dpgen#1359] [Q&A]
[train.log](https://github.com/deepmodeling/dpgen/files/12471016/train.log)

**Comment by njzjz** [comment]
The possible reason is that you have a previous unfinished job in the same directory, and never clean the remote directory. Then when you change the parameters, the program tries to read the previous checkpoint and then throws errors.
